app/services/filter_application_choices_for_providers.rb

```diff
@@ -92,6 +92,20 @@ def hide_in_reporting(application_choices, hide_in_reporting)
       application_choices.where(candidate: { hide_in_reporting: })
     end
 
+    def course_type(application_choices, course_type_filter)
+      return application_choices if course_type_filter.blank? || all_course_types?(course_type_filter)
+
+      if course_type_filter.include?('teacher_degree_apprenticeship_courses')
+        application_choices.joins(:current_course).where(current_course: { program_type: 'TDA' })
+      else
+        application_choices.joins(:current_course).where.not(current_course: { program_type: 'TDA' })
+      end
+    end
+
+    def all_course_types?(course_type_filter)
+      course_type_filter.compact_blank.sort == %w[postgraduate_courses teacher_degree_apprenticeship_courses]
```

Comment:
```suggestion
      course_type_filter.compact_blank.sort == %w[postgraduate teacher_degree_apprenticeship]
```
Is there a way to reference the constants from the `ProviderApplicationsFilter`?

Reply:
There is. I decided not do it because there would be a tightly couple. I will add